compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/ComposeSceneMediator.uikit.kt

```diff
@@ -112,29 +109,34 @@ import platform.UIKit.UIWindow
  *
  * @property rootView The UI container associated with the semantics owner.
  * @property coroutineContext The coroutine context to use for handling semantics changes.
- * @property getAccessibilitySyncOptions A lambda function to retrieve the latest accessibility synchronization options.
  * @property performEscape A lambda to delegate accessibility escape operation. Returns true if the escape was handled, false otherwise.
  */
-@OptIn(ExperimentalComposeApi::class)
 private class SemanticsOwnerListenerImpl(
     private val rootView: UIView,
     private val coroutineContext: CoroutineContext,
-    private val getAccessibilitySyncOptions: () -> AccessibilitySyncOptions,
     private val convertToAppWindowCGRect: (Rect, UIWindow) -> CValue<CGRect>,
     private val performEscape: () -> Boolean
 ) : PlatformContext.SemanticsOwnerListener {
+
     private var mediator: AccessibilityMediator? = null
 
+    var isEnabled: Boolean = false
+        set(value) {
+            field = value
+            mediator?.isEnabled = value
```

Comment:
What `mediator.isEnabled` means? Not sure when it was added...

Reply:
That's `AccessibilityMediator` class. Is enabled means that it will be providing accessibility data.

Reply:
Okay. Messed with another `mediator`. Can we name the property more specific?

Reply:
Done